utils/locales.ts

```diff
@@ -10,6 +10,12 @@ export type TranslationId = TFuncKey<'common'>;
  */
 export type CountryVariant = 'cs' | 'sk' | 'pl';
 export type Language = CountryVariant | 'uk';
+export enum LanguageEnum {
```

Comment:
Why do you need a enum when you already have `Language` ? Enums are tricky in TypeScript and can lead to unexpected bugs.

Reply:
I still don't understand why you need enum. Why not just the raw `cs` / `uk` values, type-checked against `Language`, like the rest of the codebase? 

Intead of `isActive[LanguageEnum.CZECH]`, why not just `isActive.cs` ?

Reply:
ok, removed